middleman_protocol/middleman_protocol/utils.py

```diff
@@ -0,0 +1,36 @@
+from middleman_protocol.constants import ESCAPE_SEQUENCES
+from middleman_protocol.constants import ESCAPE_CHARACTER
+from middleman_protocol.constants import FRAME_SEPARATOR
+
+
+def append_frame_separator(raw_frame: bytes) -> bytes:
+    """ Adds frame separator to raw frame. """
+    assert isinstance(raw_frame, bytes)
+
```

Comment:
We now have `helpers.py` and `utils.py` and the difference between them is pretty fuzzy. I think that functions from both should be in the same module and it should have a more concrete name. E.g. `framing.py`, `stream.py` or something like that.

Reply:
I would keep `utils.py` as they are now, but rename `helpers.py` into something more meaningful. `stream.py` sounds good. However content of `utils.py` differs very much from content of `helpers.py` so I would keep them separate. How about that ?

Reply:
But why do you want them separate? They all contain code related to converting the raw stream into messages and messages into raw stream. Splitting them like this seems pretty arbitrary to me. It's better if all the related code is gathered in a single place.

The rule that decides how they're split is not obvious and names like `helpers` and `utils` are to vague and don't give any hint that would help explain it.

Reply:
I have considered them as `helpers.py` for functions which do something with the socket, and `utils.py` for those which do something with message only (but I agreed already there should be better naming). But the way in which you have described makes sense, I will gather them in `stream.py`. :+1: done